### PR TITLE
KS-128, KS-129

### PR DESCRIPTION
**Pull request by leykov**
* Added logg in the console that the confing dependent tests failed.
 `Confing dependent tests not passed successfully.`

### CODE REVIEW

File: SDK/KountRisSdk.nuspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Kount.Net.RisSDK</id>
-    <version>6.5.0</version>
+    <version>6.5.0.1</version>
```
Comment (boyankukushev) [inline SDK/KountRisSdk.nuspec]: This should be `6.5.1`. Please check http://semver.org/ for information on semantic versioning.

File: builds/zipBuild.ps1 [hunk 2]
```diff
@@ -158,6 +158,7 @@ if ([string]::IsNullOrEmpty($TestConfig))
 	& $VsTestPath\vstest.console.exe $ConfigTestDLL /Logger:trx 
 }
 else{
+	Write-Host "Confing dependent tests not passed successfully. Please, add required settings." -foregroundcolor "red"
```
Comment (boyankukushev) [inline builds/zipBuild.ps1]: It should be `Config-dependent tests did not pass ...`

File: builds/zipBuild.ps1 [hunk 3]
```diff
@@ -158,6 +158,7 @@ if ([string]::IsNullOrEmpty($TestConfig))
...
 	$wshell = New-Object -ComObject Wscript.Shell
     $wshell.Popup($TestConfig,0,"Confing dependent tests CHECK",0)
```
Comment (boyankukushev) [inline builds/zipBuild.ps1]: `Config-dependent`

File: SDK/KountRisSdk.nuspec (hunk 1 above)
Comment (leykov) [inline SDK/KountRisSdk.nuspec]: Fixed to 6.5.1.

File: builds/zipBuild.ps1 (hunk 2 above)
Comment (leykov) [inline builds/zipBuild.ps1]: Fixed to config-dependent

File: builds/zipBuild.ps1 (hunk 3 above)
Comment (leykov) [inline builds/zipBuild.ps1]: Fixed to Config-dependent

File: SDK/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -35,5 +35,5 @@
 //
 // You can specify all the values or you can default the Revision and Build Numbers
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("6.5.0.0")]
-[assembly: AssemblyFileVersion("6.5.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("6.5.1.0")]
+[assembly: AssemblyFileVersion("6.5.1.0")]
```
Comment (boyankukushev) [inline SDK/Properties/AssemblyInfo.cs]: No .0 at the end of the versions

Comment (leykov) [inline SDK/Properties/AssemblyInfo.cs]: It doesn't play any role for versioning and Nuget package generation.
// Version information for an assembly consists of the following four values:
//
//      Major Version
//      Minor Version
//      Build Number
//      Revision

Comment (boyankukushev) [inline SDK/Properties/AssemblyInfo.cs]: Ok then ; )

**Pull request merged**